src/builders.ts

```diff
@@ -47,7 +47,7 @@ export function _buildTodoDatum(
   baseDir: string,
   lintResult: LintResult,
   lintMessage: LintMessage,
-  daysToDecay?: DaysToDecay
+  daysToDecay?: TodoConfig
```

Comment:
Can we make the variable match the updated interface name?

Reply:
Just a miss on my part.

Reply:
Fixed.